api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java

```diff
@@ -93,7 +94,8 @@ void shouldReturnTheSameIdIfOneIsSet() {
         //Given
         UUID id = UUID.randomUUID();
         //When
-        Claimant claimant = Claimant.builder().id(id).build();
+        Claimant claimant = Claimant.builder().build();
```

Comment:
Does `Claimant.builder().id(id).build();` not work any more? :(

Reply:
It doesn't unfortunately :(